cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/InMemoryStreamWriterSizeFetcher.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.data.stream.service;
+
+import co.cask.cdap.data2.transaction.stream.StreamConfig;
+
+import java.io.IOException;
+
+/**
+ * Implementation of the {@link StreamWriterSizeFetcher} for Streams which data is kept in-memory.
+ */
+public class InMemoryStreamWriterSizeFetcher implements StreamWriterSizeFetcher {
```

Comment:
No we don't, you're right. Since it is needed only by the implementations of ``StreamService``, and there is no stream service for in-memory.